src/Commands/Admin.py

```diff
@@ -151,6 +154,52 @@ def brokenQueue(player: Member, roles: List[Role]) -> Embed:
     )
 
 
+async def forceReport(mentions: str, roles: List[Role], lbChannel: Channel, *arg) -> Embed:
+    if (Queue.isBotAdmin(roles)):
+
+        if (len(arg) > 0 and "<@!" in arg[0]):
+            split = mentions.split("<@!")
+            if (len(arg) == 2 and str(arg[1]).lower() == Team.BLUE):
```

Comment:
THis `len(arg) == 2` is a bit redundant. Can we replace the `len(arg) > 0` with this instead?